packages/components/src/SectionMessage/SectionMessage.tsx

```diff
@@ -92,13 +103,18 @@ export const SectionMessage = ({
   });
   const Icon = icons[variant];
 
-  const [isVisible, setIsVisible] = useState(true);
+  const isUncontrolled = dismissable === undefined;
+  const [internalVisible, setInternalVisible] = useState(true);
+  const isCurrentlyVisible = isUncontrolled ? internalVisible : dismissable;
```

Comment:
```suggestion
  const [internalVisible, setInternalVisible] = useState(true);
  const isCurrentlyVisible = dismissable ?? internalVisible;
```
Can a liitle bit simplified.